src/styles/components/_home-search-suggestion.scss

```diff
@@ -0,0 +1,128 @@
+.home-search-suggestion {
+  &__autocomplete,
+  &__not-found {
+    all: unset;
+    position: absolute;
+    top: 105%;
+    left: 10%;
+
+    width: 80%;
+    max-height: rem(120px);
+    padding-block-end: 1rem;
+    overflow-y: auto;
+
+    box-shadow: rem(5px 10px 20px) 0 box-shadow-opacity(0.16);
+    background-color: $light;
+
+    font-size: rem(14px);
+    list-style: none;
+    text-overflow: ellipsis;
```

Comment:
Correct me if I am wrong, this part doesn't do the job, maybe you can put this `text-overflow: ellipsis;` in the `<li>` instead.